src/messages/textDocument_document.cc

```diff
@@ -130,6 +130,20 @@ template <> bool ignore(const QueryVar::Def *def) {
 }
 } // namespace
 
+bool is_macro_expansion(DB *db, WorkingFile *wf, QueryFile *file,
```

Comment:
This can be folded into `ignore` for `QueryVar::Def`

Reply:
It's a little bit cumbersome, I'll make a commit, so you can see what it looks like, but we would have to edit all `ignore` functions not just for `QueryVar::Def` because we want to ignore all symbols generated by macro expansion not just variables.